<span style="color: blue;">Deportes</span>. En un gimnasio sólo para mujeres se realizan ejercicios aeróbicos y ejercicios en aparatos. El circuito está constituido por nueve aparatos tonificadores y ocho estaciones para los ejercicios aeróbicos. En un día cualquiera, en hora pico (entre las 19 y 20 horas), llegan en promedio siete mujeres. En ese horario:

a) ¿Cuál es la probabilidad de que lleguen menos de cinco mujeres?  
b) ¿Cuál es la probabilidad de que estén ocupados todos los aparatos?  
c) Y en un día extraordinario, ¿cuál es la probabilidad de que tanto los aparatos como las estaciones estén ocupados?  
d) ¿Por qué es importante practicar algún deporte? ¿Qué beneficios trae su práctica? ¿Haces algún deporte? ¿A qué conclusiones llegas? Realiza una breve exposición con tus compañeros  



## a) probabilidad de que lleguen menos de cinco mujeres

La expresión “menos de cinco personas” significa:

$$P(X<5)$$

Utilizamos la regla del complemento:

$$\begin{align*}  P(X < 5) &= P(X \le 4) \\  &= P(X=0) + P(X=1) + P(X=2) + P(X=3) + P(X=4) \\  &= \frac{e^{-7}(7)^0}{0!} + \frac{e^{-7}(7)^1}{1!} + \frac{e^{-7}(7)^2}{2!} + \frac{e^{-7}(7)^3}{3!} + \frac{e^{-7}(7)^4}{4!} \\  &= 0.000912 + 0.006383 + 0.022341 + 0.052129 + 0.091226 \\  &= 0.172992  \end{align*}$$

por lo tanto:

$$P(X<5) = 0.172992$$

la probabilidad de que lleguen menos de cinco mujeres es aproximadamente 
$17.29\%$.

In [2]:
import numpy as np
import matplotlib.pyplot as plt
from scipy.stats import poisson
from IPython.display import HTML, display

# Parámetro de la distribución
lam = 7

# Valores que se mostrarán en la tabla y en la gráfica
x = np.arange(0, 7)

# Probabilidades P(X = x)
probabilidades = poisson.pmf(x, mu=lam)

# Construcción de la tabla HTML
html = """
<table style="
    border-collapse: collapse;
    width: 520px;
    text-align: center;
    font-family: Arial, sans-serif;
">
    <thead>
        <tr>
            <th style="
                border: 1px solid #000;
                padding: 10px;
                width: 200px;
                background-color: #f2f2f2;
            ">x (Mujeres que llegan)</th>
            <th style="
                border: 1px solid #000;
                padding: 10px;
                width: 320px;
                background-color: #f2f2f2;
            ">P(X = x)</th>
        </tr>
    </thead>
    <tbody>
"""

for valor, probabilidad in zip(x, probabilidades):
    html += f"""
        <tr>
            <td style="border: 1px solid #000; padding: 8px;">
                {valor}
            </td>
            <td style="border: 1px solid #000; padding: 8px;">
                {probabilidad:.6f}
            </td>
        </tr>
    """

html += """
    </tbody>
</table>
"""

display(HTML(html))

x (Mujeres que llegan),P(X = x)
0,0.000912
1,0.006383
2,0.022341
3,0.052129
4,0.091226
5,0.127717
6,0.149003


## b) probabilidad de que estén ocupados todos los aparatos

Asumiendo que las usuarias prefieren ocupar primero los 9 aparatos tonificadores disponibles, requerimos que lleguen exactamente 9 mujeres o mas (X >= 9) = 1 - P(X <= 8). El desglose matemático es:

$$\begin{align*} P(X \le 8) &= P(X=0) + P(X=1) + \dots + P(X=8) \\ &= 0.000912 + 0.006383 + 0.022341 + 0.052129 + 0.091226 + 0.127717 + 0.149003 + 0.149003 + 0.130377 \\ &= 0.729091 \end{align*}$$

por lo tanto:

$$P(X<8) = 27.09$$


## c) En un día extraordinario, ¿cuál es la probabilidad de que tanto los aparatos como las estaciones estén ocupados?

Para que todo el gimnasio esté lleno a su máxima capacidad (9  aparatos + 8 estaciones = 17  lugares), necesitamos que lleguen 17 o más mujeres ($X\ge17$).

$$\begin{align*}  P(X \geq 17) &= 1 - P(X \leq 16) \\  &= 1 - \left[ P(X=0) + P(X=1) + \dots + P(X=16) \right] \\  &= 1 - \left[ \frac{e^{-7}(7)^0}{0!} + \frac{e^{-7}(7)^1}{1!} + \dots + \frac{e^{-7}(7)^{16}}{16!} \right] \\  &= 1 - 0.999042 \\  &= 0.000958\end{align*}$$

## d)¿Por qué es importante practicar algún deporte?

Es importante porque ayuda con la salud a desestresarte haciendo que tu cuerpo genere dopamina y mejorando tu estado de animo

## ¿Qué beneficios trae su práctica? 

Ayuda a desestrezar liberando dopamina y mejorando el humor, ayuda a regular tu peso corporal

## ¿Haces algún deporte? 

Si salgo en bicicleta

## ¿A qué conclusiones llegas?

La practica de deportes nos ayuda a tener una mente sana y enfocar para superar retos 


<span style="color: blue;">Salud</span>. De acuerdo con la Encuesta de Consumo de Alcohol, Tabaco y otras Drogas en la Ciudad de México (2009), 25.3% de las personas consumen marihuana, drogas inhalables o cocaína. Si tenemos un grupo de 25 personas.  

a) ¿Cuál es la probabilidad de que 10 de ellas consuman alguna de estas drogas?  
b) ¿Cuál es la probabilidad de que al menos 20 de ellas consuman alguna de estas drogas?  
c) De las siguientes 50 personas, ¿cuántas esperarías que consumieran alguna de estas drogas?  
d) Actualmente, el problema de consumo de drogas se presenta en todos los estratos sociales. Por lo general se utilizan para “estar en ambiente”, o bien, para “evadir la realidad”, entre otras muchas razones. ¿Sabes las consecuencias que el consumo de drogas trae consigo? ¿Crees que las campañas para vivir sin drogas han sido eficaces? Comenta con tus compañeros qué acciones se pueden llevar a cabo para prevenir su consumo.

sea:  
Probabilidad de exito (p): 0.253  
Probabilidad de fracaso(q=1-p):0.747  
tamaño de la muestra (n): 25 personas  

formula  
$$P(X = k) = \binom{n}{k} \cdot p^k \cdot q^{n-k}$$

In [5]:
import numpy as np
import matplotlib.pyplot as plt
from scipy.stats import binom
from IPython.display import HTML, display

# Parámetros de la distribución binomial actualizados
n = 25
p = 0.253

# Valores de x (de 0 a n)
x = np.arange(0, n + 1)

# Probabilidades P(X = x)
probabilidades = binom.pmf(x, n=n, p=p)

# Construcción de la tabla HTML
html = """
<table style="
    border-collapse: collapse;
    width: 520px;
    text-align: center;
    font-family: Arial, sans-serif;
">
    <thead>
        <tr>
            <th style="
                border: 1px solid #000;
                padding: 10px;
                width: 200px;
            ">x</th>
            <th style="
                border: 1px solid #000;
                padding: 10px;
                width: 320px;
            ">P(X = x)</th>
        </tr>
    </thead>
    <tbody>
"""

for valor, probabilidad in zip(x, probabilidades):
    html += f"""
        <tr>
            <td style="border: 1px solid #000; padding: 8px;">
                {valor}
            </td>
            <td style="border: 1px solid #000; padding: 8px;">
                {probabilidad:.6f}
            </td>
        </tr>
    """

html += """
    </tbody>
</table>
"""

display(HTML(html))

x,P(X = x)
0,0.000681
1,0.005764
2,0.023428
3,0.060833
4,0.113320
5,0.161196
6,0.181984
7,0.167297
8,0.127489
9,0.081560


## a) ¿Cuál es la probabilidad de que 10 de ellas consuman alguna de estas drogas?

$$P(X = 10) = \binom{25}{10} \cdot (0.253)^{10} \cdot (0.747)^{15}$$  

$$P(X = 10) = 3,268,760 \cdot (9.755 \times 10^{-7}) \cdot 0.01235 \approx 0.0394$$

la probabilidad aproximada es de: 3.94%

## b) ¿Cuál es la probabilidad de que al menos 20 de ellas consuman alguna de estas drogas?
$$P(X \ge 20) = P(X=20) + P(X=21) + P(X=22) + P(X=23) + P(X=24) + P(X=25)$$

las probabilidades son muy bajos por que el exito es de 25.3% esta muy lejos de 20 personas de un grupo de 25 por ejemplo x=20

$P(X=20) \approx 2.44 \times 10^{-8}$  
$P(X=21) \approx 2.12 \times 10^{-9}$  
$P(X=22) \approx 1.44 \times 10^{-10}$  
$P(X=23) \approx 6.94 \times 10^{-12}$  
$P(X=24) \approx 2.35 \times 10^{-13}$  
$P(X=25) \approx 3.98 \times 10^{-15}$  

sumamos desde 20 a 25 el resultado es
$$P(X \ge 20) \approx 2.67 \times 10^{-8} \approx 0.0000000267$$

## c) De las siguientes 50 personas, ¿cuántas esperarías que consumieran alguna de estas drogas?

Para calcular el valor esperado usamos la fórmula:$$\mu = n \cdot p$$

entonces:

$$\mu = 50 \cdot 0.253 = 12.65$$

es decir que aproximadamente 13 personas consumen droga

## d) ¿Sabes las consecuencias que el consumo de drogas trae consigo?

provocan daños neurológicos reversibles, problemas cardiovasculares, trastornos mentales y dependencia física y psicológica.

## ¿Crees que las campañas para vivir sin drogas han sido eficaces?

las campañas basadas puramente en el miedo o la prohibición han mostrado eficacia limitada a largo plazo